### DISCUSSION TITLE
Is it possible to add a nolint directive in the go.mod file?

### DISCUSSION THREAD

**Discussion by dopey** [golangci/golangci-lint#3252] [Q&A] [answered]
For example to add a `//nolint` to a `replace` directive.

**Comment by ldez** [comment, marked as answer]
Hello,

It's not possible, why do you need that?

If it's related to gomoddirective, I recommend using the configuration of the linter:
https://golangci-lint.run/usage/linters/#gomoddirectives

**Reply by dopey** [reply]
Hey @ldez, thanks for the quick reply! I am using the gomoddirective configuration in the linter config, however I also use the same golangci.yml across my entire organization. I don't want to maintain separate versions of that file for every repo because that's been a great way to get out of sync. I want consistent lint-int everywhere. Anyway, it would be nice not to have to add new exclusions to the global configuration for each repo and rather just handle the exclusions in the code itself (as I would with other linter warnings).

**Reply by ldez** [reply]
IMO it's not a proper approach to play with the nolint directives instead of using a specific configuration.
The nolint directives work only inside Go code.
gomoddirective is a specific linter because the target of this linter is not Go code but the go mod.

**Reply by dopey** [reply]
Fair enough. Not sure I agree with the verdict, but I appreciate the response and respect the decision.
